keyboards/giabalanai/giabalanai.h

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2020 3araht
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
+    l00, l01, l02, l03, l04, l05, \
+    l10, l11, l12, l13, l14, l15, \
+    l20, l21, l22, l23, l24, l25, \
+    l30, l31, l32, l33, l34, l35, \
+    l40, l41, l42, l43, l44, l45, \
+    \
+    l06, l07, l08, l09, l0a, l0b, \
+    l16, l17, l18, l19, l1a, l1b, \
+    l26, l27, l28, l29, l2a, l2b, \
+    l36, l37, l38, l39, l3a, l3b, \
+    l46, l47, l48, l49, l4a, l4b, \
```

Comment:
Since I have no idea what this keyboard actually looks like, based on your keymap, it looks like this is closer to the actual, physical layout: 

```suggestion
    l00, l01, l02, l03, l04, l05,     l06, l07, l08, l09, l0a, l0b, \
    l10, l11, l12, l13, l14, l15,     l16, l17, l18, l19, l1a, l1b, \
    l20, l21, l22, l23, l24, l25,     l26, l27, l28, l29, l2a, l2b, \
    l30, l31, l32, l33, l34, l35,     l36, l37, l38, l39, l3a, l3b, \
    l40, l41, l42, l43, l44, l45,     l46, l47, l48, l49, l4a, l4b, \
    \
```

Reply:
You're right. The physical layout looks what you specified.
The problem with it was the actual assignment for each keys end up having too long rows in keymap.c.

I'd like to follow your suggestions which way to go, whether:
  Apply your modifications here and have long rows in keymap.c 
or 
  Keep it as it is to avoid long rows in keymap.c

Thanks!

Reply:
Long rows are fine.  And we'd/I'd prefer that the layout in the keymap.c files actually resemble what the board looks like, as it's easier to visualize for a lot of people.

Reply:
Roger that.
I'll update the header file.

Reply:
@drashna, could you "Resolve conversation" since it is done?
Thanks!